assets/charts/control-plane/kubelet/container-image/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM quay.io/poseidon/kubelet:v1.20.4
+FROM quay.io/poseidon/kubelet:v1.20.5
```

Comment:
This image has to be built and pushed for the CI to pass.

Reply:
Is pushed now :)